test/test_ops.py

```diff
@@ -436,6 +475,12 @@ def test_msroialign_repr(self):
         )
         assert repr(t) == expected_string
 
+    @pytest.mark.parametrize("device", cpu_and_gpu())
+    def test_feature_extractor(self, device):
+        op_obj = TestMultiScaleRoIAlign.make_obj().to(device=device)
```

Comment:
same here, unclear why we do static calls if the base-class defines `make_obj()` as a non-static method.